rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -318,7 +318,8 @@ public void registerBtcTransaction(Transaction rskTx, byte[] btcTxSerialized, in
         }
 
         // Check there are at least N blocks on top of the supplied height
-        int confirmations = btcBlockChain.getBestChainHeight() - height + 1;
+        int btcBestChainHeight = getBtcBlockchainBestChainHeight();
+        int confirmations = btcBestChainHeight - height + 1;
         if (confirmations < bridgeConstants.getBtc2RskMinimumAcceptableConfirmations()) {
             logger.warn(
                     "At least {} confirmations are required, but there are only {} confirmations",
```

Comment:
add +1 to confirmations in the log

Reply:
Commented on the corresponding issue: https://rsklabs.atlassian.net/browse/RFS-183

Reply:
good point